LEARNING.md

```
@@ -0,0 +1,45 @@
+
+## Mission
+Research to understand AI engineering fundamentals and explore what to build
+
+## Placement
+- Date: 2026-08-29
+- Score: self-selected
+- Entry point: Phase 0: Setup & Tooling
+- Pace: ~5 hours/week
+
+## Path
+| Phase | Name | Status | Est. hours |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)